<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogReminder</class>
 <widget class="QDialog" name="DialogReminder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Restart Required</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SystemUpdaterTray.qrc">
    <normaloff>:/images/restart.png</normaloff>:/images/restart.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox::title{
  subcontrol-origin: none;
}
QGroupBox{
background: rgba(255, 255, 127, 150);
border: 1px solid grey;
border-radius: 5px;
}</string>
     </property>
     <property name="title">
      <string notr="true"/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="pixmap">
         <pixmap resource="SystemUpdaterTray.qrc">:/images/restart.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>The system needs to be restarted to finish installing updates.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Next Reminder:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Never</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30 Minutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 Hour</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 Hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12 Hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24 Hours</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="tool_reboot">
       <property name="text">
        <string>Reboot Now!</string>
       </property>
       <property name="icon">
        <iconset resource="SystemUpdaterTray.qrc">
         <normaloff>:/images/restart.png</normaloff>:/images/restart.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="tool_close">
       <property name="text">
        <string>Remind Me Later</string>
       </property>
       <property name="icon">
        <iconset resource="SystemUpdaterTray.qrc">
         <normaloff>:/images/view-refresh.png</normaloff>:/images/view-refresh.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="SystemUpdaterTray.qrc"/>
 </resources>
 <connections/>
</ui>
